coding-concepts.js

```diff
@@ -56,4 +68,7 @@ const dataTypes = ["number", "string", "Boolean"]
 // console.log(typeof dataTypes.length)
 
 // a) Your answer:
+// I believe that the output will be undefined. Since typeof dataTypes was no defined as a variable.
+
 // b) Verify and explain:
+// The output was just number. I believe it returned number because .length shows only the numerical output by calculating the number of places of index that has been filled plus 1. But typeof indicates the type of varialble in your code. Thus, by removing .length it should return something different which was object.
```

Comment:
Close. The first thing executed is dataTypes.length which produce 3, and typeof 3 will output number. In other words, 
`dataTypes.length` evaluates the amount of values in the array, which is 3 then 
typeof evaluates 3, which is a number datatype.